src/services/users.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+let users = [];
+
+function getAllUsers() {
+  return users;
+}
+
+function getUser(userId) {
+  return users.find(user => user.id === Number(userId));
+}
+
+function createNewUser(name) {
+  let newId = 0;
+
+  if (users.length) {
+    newId = [...users].sort(
+      (userA, userB) => userB.id - userA.id
+    )[0].id + 1;
```

Comment:
you can create folder utils and move repeated code there, for finding correct id you can create a function